server/src/routes/serializers/ReviewSerializer.js

```diff
@@ -0,0 +1,17 @@
+import UserSerializer from "./UserSerializer.js"
+
+class ReviewSerializer {
+  static async getSummary(review) {
+    const allowedAttributes = ["id", "comment", "rating"]
+
+    let serializedReview = {}
+    for (const attribute of allowedAttributes) {
+      serializedReview[attribute] = review[attribute]
+    }
+    const relatedUser = await review.$relatedQuery("users")
```

Comment:
It looks like we have a relationMapping of `user` but not `users` on the `Review.js` model.

We could have
```js
const relatedUser = await review.$relatedQuery("user")
```